semaphoreci/semaphore-runner.sh

```diff
@@ -37,7 +37,6 @@ sed 's/^deb/deb-src/' /etc/apt/sources.list >> /etc/apt/sources.list.d/sources.l
 while [ -z "\$(ip route list 0/0)" ]; do sleep 1; done
 apt-get -q --allow-releaseinfo-change update
 apt-get -y dist-upgrade
-apt-get install -y eatmydata
```

Comment:
I don't see how this could make a difference. Fsync is about making the in-memory and on-disk states consistent, and we don't care about the on-disk state at all, since we're only using an ephemeral machine.

Reply:
Hm, my train of though was that even though the machine is ephemeral, it's still backed by a "hard" storage (`/var/lib/lxc` in this case), so that in combination with ~4G of memory (not sure if in all cases) could cause the mentioned gcc issues. 

But I guess I'm missing a puzzle piece. somewhere